marlowe-symbolic/src/Language/Marlowe/Analysis/FSSemantics.hs

```diff
@@ -39,45 +80,65 @@ generateSymbolicInterval (Just ms) =
      constrain (ls .>= literal ms)
      return i
 
+-- foldWithKey for Plutus' AssocMap
```

Comment:
are plutus planning on adding stuff like this?

Reply:
You know you can just add stuff to our libraries, right? :)

Reply:
oh cool. *cough @palas cough*